skl2onnx/common/_apply_operation.py

```diff
@@ -138,17 +138,30 @@ def apply_cast(scope, input_name, output_name, container, operator_name=None,
         v.number: k for k, v in d.values_by_name.items()
     }
     if to not in allowed_type_name_and_type_enum_pairs:
-        raise ValueError('Attribute to must be one of %s'
```

Comment:
It is usually "Attribute 'op' must...

Reply:
PEP8 doesn't recommend one over the other. Any particular reason for your preferance?

Reply:
PEP8 doesn't recommend one over the other. Any particular reason for your preferance?